src/main/flush.ts

```diff
@@ -64,6 +73,32 @@ export const save = (state: IState) => {
   }
 };
 
+export const fetchPreset = (preset_name: string) => {
+  try {
+    const content = fs.readFileSync(PRESETS_DIR + preset_name, {
+      encoding: 'utf8',
+    });
+    return JSON.parse(content) as IPreset;
+  } catch (ex) {
+    console.log('Failed to get presets!!');
+    throw ex;
+  }
+};
+
+export const savePreset = (preset_name: string, preset_info: IPreset) => {
+  // need to create presets dir first...... should we do this in the install?
```

Comment:
I think this comment isn't necessary anymore now that we're creating a presets dir right? Let's remove it if we can.